modules/tensor_mechanics/doc/content/getting_started/NEML2.md

```diff
@@ -0,0 +1,75 @@
+# NEML2
+
+In addition to the MOOSE native material models, MOOSE can also seamlessly interface with the external material modeling library [NEML2](https://github.com/reverendbedford/neml2) [!cite](neml2_anl_report).
+
+## What is NEML2?
+
+NEML2 stands for the New Engineering Material model Library, version 2.
+
+NEML2 is an offshoot of [NEML](https://github.com/Argonne-National-Laboratory/neml),
+an earlier constitutive modeling code developed at Argonne National Laboratory that
+can be used through the MOOSE based [BlackBear](https://github.com/idaholab/blackbear) application.
```

Comment:
are you referring to NEML here or NEML2. If 2 then say the TM module instead

Reply:
Is that not clear from the context? NEML2, which is still only available through BlackBear.

Reply:
NEML2 is available through TM from now on?

Reply:
"NEML2 is an offshoot of NEML (which can be used though Blackbear). "

I see how this could be read both ways (you seem to get from it that NEML2 is available through Blackbear, but the "that" refers to NEML (one). I'll clarify this.